### PR TITLE
add PR approval process

### PR DESCRIPTION
**Pull request by xmulligan**
Following up on the discussion from #74

### CODE REVIEW

File: charter.md [hunk 1]
```diff
@@ -172,3 +172,11 @@ If insufficient members resign their roles, the positions held by members associ
 #### Effect on representation
 
 After any election, the results may indicate that a particular entity is overrepresented for a role, having won more seats than the limit.  In this event, the maximum number of the members belonging to that entity shall be calculated, and members belonging to the entity having the least votes will be excluded from the role until the entity is fairly represented.  Where there is a tie, a coin toss or other random selection mechanism shall be used to choose who is allocated the seat.
+
+## PR Approval Process
+
+Use cases and best practices need approval from at least two tech leads to be merged. Approval from three community members can substitute for one tech lead vote. Merging may be veoted by the co-chairs with a vote of at least 2/3.
```
Comment (jeffsaelens) [inline charter.md]:
```suggestion
Use cases and best practices need approval from at least two tech leads to be merged. Approval from three community members can substitute for one tech lead vote. Merging may be vetoed by the co-chairs with a vote of at least 2/3.
```

File: charter.md [hunk 2]
```diff
@@ -172,3 +172,11 @@ If insufficient members resign their roles, the positions held by members associ
...
+
+Charter updates need approval from at least two co-chairs. A co-chair vote can be replaced by the approval of two tech leads.
+
+Changes to fix broken links, typos, small gramatical phrasing, or other editorial issues can be merged with the approval any co-chair or tech lead.
```
Comment (jeffsaelens) [inline charter.md]:
```suggestion
Changes to fix broken links, typos, small grammatical phrasing, or other editorial issues can be merged with the approval of any co-chair or tech lead.
```

**Review by jeffsaelens — CHANGES_REQUESTED** [review]
A couple of quick fixes.

**Comment by xmulligan** [comment]
@jeffsaelens thanks for the grammar updates, should have done it in a word doc first 😅

**Review by iawells — APPROVED** [review]

File: charter.md [hunk 3]
```diff
@@ -172,3 +172,11 @@ If insufficient members resign their roles, the positions held by members associ
 #### Effect on representation
 
 After any election, the results may indicate that a particular entity is overrepresented for a role, having won more seats than the limit.  In this event, the maximum number of the members belonging to that entity shall be calculated, and members belonging to the entity having the least votes will be excluded from the role until the entity is fairly represented.  Where there is a tie, a coin toss or other random selection mechanism shall be used to choose who is allocated the seat.
+
+## PR Approval Process
+
+Use cases and best practices need approval from at least two tech leads to be merged. Approval from three community members can substitute for one tech lead vote. Merging may be vetoed by the co-chairs with a vote of at least 2/3.
```
Comment (jeffsaelens) [inline charter.md:178]: Need to establish specifically what a tech lead is. Is it one tech lead or both that can be replaced by 3 community votes?

File: charter.md [hunk 4]
```diff
@@ -172,3 +172,11 @@ If insufficient members resign their roles, the positions held by members associ
...
+
+Charter updates need approval from at least two co-chairs. A co-chair vote can be replaced by the approval of two tech leads.
```
Comment (jeffsaelens) [inline charter.md:180]: @fkautz "This is somewhat of a low bar for replacing the co-chair"

**Review by jeffsaelens — CHANGES_REQUESTED** [review]
During the WG call, it was discussed that we need more clarity on roles.

**Comment by taylor** [comment]
See discussion for minimal process with delegation to community on all but a few items https://github.com/cncf/cnf-wg/discussions/130

**Comment by taylor** [comment]
@xmulligan are you willing to close this PR out in favor of https://github.com/cncf/cnf-wg/pull/131

File: charter.md [hunk 5]
```diff
@@ -172,3 +172,11 @@ If insufficient members resign their roles, the positions held by members associ
...
+
+Changes to fix broken links, typos, small grammatical phrasing, or other editorial issues can be merged with the approval of any co-chair or tech lead.
```
Comment (electrocucaracha) [inline charter.md:182]: This last criteria covers the rest of the cases so what about something like 
```suggestion
Any other change like fix broken links, typos, small grammatical phrasing, or other editorial issues can be merged with the approval of any co-chair or tech lead.
```

**Comment by xmulligan** [comment]
@taylor I'll leave it open until one or the other is merged so people have ideas for different options

**Comment by taylor** [comment]
> @taylor I'll leave it open until one or the other is merged so people have ideas for different options

@xmulligan close?  re: #131

**Pull request closed without merging**